lua/zipkin.lua

```diff
@@ -70,36 +70,34 @@ end
 -- microseconds.
```

Comment:
I think this line is out of date. It should mention the fact that they are logged always, regardless of sampling decision.

Reply:
Reading this comment I just realized I need to check whether the headers are present or not. Before that was done implicitly since `nil != "1"`